### ISSUE TITLE
consistent arch diagrams and images in docs 

### ISSUE THREAD

**Issue by peterj** [istio/istio.io#15195]
At some point we should decide on a "style" and re-draw the diagrams across the docs, so they are all consistent and look & feel as they are part of the same product.

**Comment by craigbox** [comment]
https://drive.google.com/file/d/1YRY7TYSfrkAbpHuYPjdeHUwZMMzDlPvh/view

**Comment by peterj** [comment]
Yep, I don't mind the look & feel, it's mostly about staying consistent throughout the documentation and updating the diagrams that don't fit the same look and feel. For example:

![Screenshot 2024-06-03 at 1 55 05 PM](https://github.com/istio/istio.io/assets/11080940/a0423ae1-7747-4316-a7bd-8a7e9e2ed212)

![Screenshot 2024-06-03 at 1 55 57 PM](https://github.com/istio/istio.io/assets/11080940/3b7c0709-8b42-44ca-a29c-d52bf3da4eb6)

![Screenshot 2024-06-03 at 1 56 36 PM](https://github.com/istio/istio.io/assets/11080940/ab6a9ce4-ba8e-4149-b7cb-ef2747387cf9)

**Comment by craigbox** [comment]
(No disrespect to the authors, but all three are butt ugly, and I can point you to a dozen more!)

I don't think Excalidraw is going to cut it btw - I'm thinking we need Illustrator templates or something like Canva.

**Comment by craigbox** [comment]
We should optimize for whatever is the most likely to deliver consistent, nice looking output, that any contributor can easily access.

Then I can put a manual lint gate on people who don't submit drawings that look like the rest of the site in their blog posts 😁

(Maybe Google Slides isn't a bad choice?)

**Comment by craigbox** [comment]
It's working for me so far.

https://docs.google.com/presentation/d/1p1bMhB6hIw9bucb1qeiXRXwxODZlrXI2-Gjtv_b9x94/edit#slide=id.g2ec15086a55_0_0

I think the next step would be to identify the things we want to be able to distinguish between (i.e. a colour that means data plane traffic and a colour that means control plane/configuration traffic) and then put together an upgrade to the diagram components guide.

**Comment by peterj** [comment]
I'll update the existing creation guidelines according to the new design (https://istio.io/latest/docs/releases/contribute/diagrams/). 

From there, we can fan out and have multiple people (if needed) help out with re-creating the diagrams based on these guidelines

**Comment by ilrudie** [comment]
[mermaid](https://mermaid.live/edit#pako:eNpVjr2OwkAMhF_FcsVJ5AVSIB3JQYMEEnRZCitxsivYHzkbIZTk3W9DmjtX9sw3I49Y-4Yxx_bpX7UmiXArlYM031WhxfTRUn-HLNtNR45gveP3BPvN0UOvfQjGdV8rv18gKMbTgjFEbdxjXq3ikz87nqCsThSiD_e_zu3lJ_ipzEWn-v-OFk6pQ9VS3lJWk0BB8kFwi5bFkmnS--OiKIyaLSvM09qQPBQqNyeOhuivb1djHmXgLYofOo2p8NmnawgNRS4NdUJ2ReZfufpaQg) maybe? Just brainstorming
